patches/attendance/styles.css

```diff
@@ -6,6 +6,7 @@
     border: 0;
     cursor: pointer;
     border-radius: 25px;
+    font-family: "WorkSans", Graphik, -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
so much better

Reply:
I just used same fonts as them for everything else, so it's more blending in with the rest rn